changelog/22.0/22.0.0/summary.md

```diff
@@ -25,4 +28,12 @@ For planned reparents, hosts taking backups with a backup engine other than `bui
 valid candidates. This means they will never get promoted - not even if there's no other candidates.
 
 Note that behavior for `builtin` backups remains unchanged: a replica that is currently taking a `builtin` backup will
-never be promoted, neither by planned nor by emergency reparents.
\ No newline at end of file
+never be promoted, neither by planned nor by emergency reparents.
+
+## <a id="minor-changes"/>Minor Changes
+
+### <a id="vttablet"/>VTTablet
+
+#### <a id="vttablet-consolidator-max-query-wait"/>--consolidator-max-query-wait flag
+
+A new CLI flag `--consolidator-max-query-wait` to set the maximum wait per query for consolidator. The default value is set to 0 for unlimited wait. Users can adjust this value based on the performance of VTTablet to avoid excessive memory usage and the risk of being OOMKilled, particularly in Kubernetes deployments.
```

Comment:
Is this supposed to be "maximum number of clients waiting for this query", or "maximum amount of time to wait for this query"? I understand which one it is by looking at code, but users should be able to understand it from only the flag name and help. I'll make a suggestion for it where the flag is defined.

Reply:
updated per your comment, thanks @deepthi !